lib/addons/p5.dom.js

```diff
@@ -1411,11 +1411,12 @@
    */
   p5.Element.prototype.addClass = function(c) {
     if (this.elt.className) {
-      // PEND don't add class more than once
+      // PEND don't add class more than once*
```

Comment:
Why is the extra character added here at the end?

Reply:
Don't mind, just mistype when checked my code. It will be removed. @limzykenneth should I remove comment as well? I think it is reasonable because this comment about my changes.

Reply:
The few lines of comments here, including the commented out code seems a bit messy and maybe not in consideration anymore if your PR is merged? If so I think it's fine to remove them and if your code need some explanation for future developers then do add some comments in there.